src/backend/metal/src/command.rs

```diff
@@ -1543,11 +1539,15 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
                         width: (ext.width >> level) as _,
                         height: (ext.height >> level) as _,
                     }),
-                    soft::RenderCommand::BindBuffer {
+                    soft::RenderCommand::BindBufferData {
                         stage: pso::Stage::Vertex,
                         index: 0,
-                        buffer: Some(vertex_buffer),
-                        offset: 0,
+                        bytes: unsafe { 
+                            slice::from_raw_parts(
+                                list.as_ptr() as *const u8, 
+                                list.len() * mem::size_of::<BlitVertex>()
```

Comment:
I think this size is too big, seems like it should be `list.len() * mem::size_of::<BlitVertex>() / mem::size_of::<u32>()` if I understand correctly